src/clj/puppetlabs/services/jruby/jruby_puppet_service.clj

```diff
@@ -84,8 +84,8 @@
    (.compileCatalog jruby-instance request-options))
 
   (compile-ast
-    [this jruby-instance compile-options boltlib-path]
-    (.compileAST jruby-instance compile-options boltlib-path))
+    [this jruby-instance compile-options]
+    (.compileAST jruby-instance compile-options))
```

Comment:
It looks like this was added as part of apply work, regardless of projects. https://github.com/puppetlabs/puppetserver/commit/e05b75e3635ae7fcbe63d295651952e40a979c3d Is that right?

Reply:
UGH, DAMNIT. 

Yeah, you're correct